src/ui/public/notify/notify.less

```diff
@@ -77,6 +77,14 @@
     }
   }
 
+  .toast-banner {
+    // no flex
+  }
+
+  .toast-message-banner {
+    display: block;
+  }
```

Comment:
It's up to you, but someone's going to have to unnest it someday. Since it's new code, I'd prefer to unnest it now to avoid "adding to the mess".